check402.ml

```diff
@@ -1,6 +1,7 @@
 let () =
+  set_binary_mode_out stdout;
```

Comment:
Hi @hhugo, thanks for submitting this PR. This code triggers the following warning at compilation:
```
  set_binary_mode_out stdout;
      ^^^^^^^^^^^^^^^^^^^^^^^^^^
Warning 5 [ignored-partial-application]: this function application is partial,
```
Could you motivate your change, I know very little about Windows. Moreover, using text mode for I/O would, in  my naive view, be more compatible that binary I/O.

Reply:
Sorry, it was a half a change from a previous iteration.
The issue with the script is that in presence of automatic newline translation on windows, the check in handle402.sh fail because there is now a trailing `\r`.
To fix it, one need to either set `binary_mode` for `stdout` or remove the newline (which this PR implement)